s3-source-connector/src/main/java/io/aiven/kafka/connect/s3/source/output/ByteArrayWriter.java

```diff
@@ -35,18 +37,33 @@ public void configureValueConverter(final Map<String, String> config, final S3So
 
```

Comment:
There needs to be some documentation or comment here explaining why the ByteArrayWriter has an empty implementation method here.